data.py

```diff
@@ -58,13 +59,153 @@ def jitter_pointcloud(pointcloud, sigma=0.01, clip=0.05):
     return pointcloud
 
 
+class Jitter:
+    def __init__(self, sigma=0.01, clip=0.05):
+        self._sigma = sigma
+        self._clip = clip
+
+    def __call__(self, pointcloud):
+        return jitter_pointcloud(pointcloud, self._sigma, self._clip)
+
+
+class RandomRemove:
```

Comment:
This is the class that applies RandomRemoval of point clouds with an arbitrary probability

@arvind will be in charge of running the experiments and validation